### PR TITLE
Cyber Triage --> PY3

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Related Issues
Py3 Hackathon

### CODE REVIEW

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+USE_PROXY = CT_PARAMS['use_proxy']
+
+
+def IS_2XX(x):
+    return (x / 100) == 2  # Returns true if status code (int) is 2xx
+
+
+# Delete request proxy environment variables if user does not want to use proxy
+if not USE_PROXY:
+    del os.environ['HTTP_PROXY']
```
Comment (tkatzir) [inline Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:33]: This proxy env deletion appears numerous times in the code.
I guess it belongs in CommonServerPython maybe?

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    http_status_code = -1
+    e = None
+
+    # Get the request function associate with the type of http request we want to make (ex. GET or POST)
+    try:
+        request_func = getattr(requests, method)
+    except AttributeError:
+        ret_msg = f'Invalid requests method: {method}'
+        return_error(ret_msg)
+    else:
```
Comment (tkatzir) [inline Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:71]: There's no `finally` in this `try-except`, so why do we need `else`?

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            'response': ret_msg,
+            'status_code': http_status_code,
+            'exception': e
+        }
+
+# Format non 2xx responses from make_rest_call(). These are cases where an error has occurred.
+
+
+def format_err_resp(json_resp):
+    return 'Error message: {}\nHTTP Status code: {}\nExceptions: {}'.format(json_resp['response'],
```
Comment (tkatzir) [inline Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:125]: Can this be changed to the `f'foo {bar}'` format?

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+REST_PORT = CT_PARAMS['rest_port']
+API_KEY = CT_PARAMS['api_key']
+USER = CT_PARAMS['credentials']['identifier']
+PASSWORD = CT_PARAMS['credentials']['password']
+VERIFY_SERVER_CERT = False
+REQ_HEADERS = {'restApiKey': API_KEY}
+USE_PROXY = CT_PARAMS['use_proxy']
+
+
+def IS_2XX(x):
```
Comment (tkatzir) [inline Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:27]: Why isn't this method called `is_2xx` (lowercase)?
No need to decorate it with `@staticmethod`?

**Review by tkatzir — APPROVED** [review]
Approved, with some comments

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py (hunk 4 above)
Comment (avidan-H) [reply Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:27]: It's just a function - not a method of a class - therefore no need.

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py (hunk 2 above)
Comment (avidan-H) [reply Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:71]: `finally` is intended primarily for cleanup whereas the `else` clause of a `try-except` block is logic flow where that is what happens if an error _doesn't_ occur.

File: Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py (hunk 1 above)
Comment (avidan-H) [reply Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py:34]: I'll replace it with `handle_proxy` from `CommonServerPython`.

**Pull request merged**